index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <h1>Hello Microverse!</h1>
+  </head>
```

Comment:
- [ ] [OPTIONAL] I would suggest not writing `h1` tag inside the `head` tag it should be inside `body` tag like:

```

<body>
<h1>Hello Microverse!</h1>
</body>

```

> The <head> element is a container for metadata (data about data) and is placed between the <html> tag and the <body> tag.

> The <body> tag defines the document's body. The <body> element contains all the contents of an HTML document, such as headings, paragraphs, images, hyperlinks, tables, lists, etc.